dataland-frontend/tests/e2e/fixtures/common/NumberFixtures.ts

```diff
@@ -8,7 +8,7 @@ import { faker } from "@faker-js/faker";
  * @returns a random number in [min, max]
  */
 export function generateFloat(min = 0, max = 1e5, precision = 1e-2): number {
-  return faker.number.float({ min: min, max: max, precision: precision });
+  return faker.number.float({ min: min, max: max, multipleOf: precision });
```

Comment:
Why the change? I have something about some faker functionality being deprecated in the back of my mind. Was it this?

Reply:
yes, while running the devtools this warning came up 5000 times

Reply:
😄